api/v1alpha1/target.go

```diff
@@ -96,25 +97,26 @@ func (s TargetSpec) DatabaseURL(ctx context.Context, r client.Reader, ns string)
 
 // URL returns the URL for the database.
 func (c *Credentials) URL() *url.URL {
+	r := regexp.MustCompile("\t|\n|\r")
```

Comment:
move to out of this function
```suggestion
	r := regexp.MustCompile("[\t\r\n]")
```